gns3/modules/vmware/dialogs/vmware_vm_wizard.py

```diff
@@ -47,7 +47,10 @@ def __init__(self, vmware_vms, parent):
             # we want to see the cancel button on OSX
             self.setOptions(QtWidgets.QWizard.NoDefaultButton)
 
-        if VMware.instance().settings()["use_local_server"]:
+            # Fusion is not supported on OSX
+            self.uiLocalRadioButton.setEnabled(False)
+
+        if Servers.instance().isNonLocalServerConfigured() is False:
```

Comment:
VMware can only be used on a local server or remote. If the GNS3 VM is enabled and not remote server configured, we get the first page where we can only select the local server option.